src/modules/users/components/Inbox/InboxContainer/InboxContainer.tsx

```diff
@@ -42,6 +42,12 @@ const MSG = defineMessages({
     id: 'users.Inbox.InboxContainer.seeAll',
     defaultMessage: 'See All',
   },
+  noItems: {
+    id: 'users.Inbox.InboxContainer.noItems',
+    defaultMessage:
+      /* eslint-disable-next-line max-len */
+      'It looks like you don’t have any notifications. Don’t worry, we’ll let you know when anything important happens.',
```

Comment:
I think we've been using template literals for longer blocks of text everywhere else, along with `'` for apostrophes. We should probably stay consistent with those

Reply:
Good call on the apostrophes! 

I just copy-pasted this from the original issue and didn't give it a second thought.